ray-operator/controllers/ray/common/pod.go

```diff
@@ -513,6 +513,13 @@ func setContainerEnvVars(pod *v1.Pod, rayContainerIndex int, rayNodeType rayiov1
 		portEnv := v1.EnvVar{Name: RAY_PORT, Value: headPort}
 		container.Env = append(container.Env, portEnv)
 	}
+	if createdByRayService {
```

Comment:
I think thinking if there's better way to indicate a cluster is created by RayServe controller. 

Can we leverage https://github.com/ray-project/kuberay/blob/50d4782f7f618290aec2bbccb0ade0d001b0bb6f/ray-operator/controllers/ray/common/constant.go#L29 and pass into the method. 

This help us not expand method arguments if we need some customization on clusters created by RayJob etc.

Reply:
sg